### PR TITLE
Update to Cloud Firestore Emulator v1.4.5

### PR DESCRIPTION
**Pull request by yuchenshi**

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -2,3 +2,5 @@ fixed - Fixes a bug in Cloud Firestore emulator where rulesets with no-op writes
 fixed - Reduced lock contention in Cloud Firestore emulator during concurrent writes to a document.
 feature - Serving Hosting locally can now proxy to the live Cloud Run service.
 fixed - Fixes a bug where fields in Cloud Firestore indexes would be sorted alphabetically for display.
+fixed - Fixes a bug for Firestore REST API.
```
Comment (ryanpbrewster) [inline changelog.txt]: please specify that this is for the emulator

Comment (ryanpbrewster) [reply changelog.txt]: and let's add a bit more detail here, this was an issue where the REST api would yield 404s to root collections

File: src/emulator/constants.js [hunk 2]
```diff
@@ -26,9 +26,9 @@ const _emulators = {
     stdout: null,
     cacheDir: CACHE_DIR,
     remoteUrl:
-      "https://storage.googleapis.com/firebase-preview-drop/emulator/cloud-firestore-emulator-v1.4.4.jar",
-    expectedSize: 56904597,
-    expectedChecksum: "b64aa203304f231b61ad7c30316d1094",
+      "https://storage.googleapis.com/firebase-preview-drop/emulator/cloud-firestore-emulator-v1.4.5.jar",
+    expectedSize: 56840576,
+    expectedChecksum: "7fa4ebe4615650038c79ea260b0e7bf6",
     localPath: path.join(CACHE_DIR, "cloud-firestore-emulator-v1.4.4.jar"),
```
Comment (bkendall) [inline src/emulator/constants.js]: I assume this version needs to be incremented.

Please test this change after you make it.

Comment (yuchenshi) [reply src/emulator/constants.js]: Good catch! I've fixed this and manually tested ti.

**Pull request merged**